tools/EMLassemblyline/table_template.xml

```diff
@@ -0,0 +1,45 @@
+<tool id="table_template" name="Make templates for data tables" version="0.1.0+galaxy0">
+    <description>attributes and categorical variables for EML metadata creation</description>
+    <requirements>
+        <requirement type="package" version="4.3.1">r-base</requirement>
+        <requirement type="package" version="3.5.5">r-emlassemblyline</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    
+    #set $table = ""
+    mkdir data_files && 
+    #for $indata in $inputdata
+        ln -s '$indata' 'data_files/${indata.element_identifier}';
+        #set $table += $indata.element_identifier + " ";
+    #end for
+    
+    Rscript 
+         '$__tool_directory__/table_templates.R'
+         '$table'
+    
+    ]]></command>
+    <inputs>
+         <param name="inputdata" type="data_collection"  collection_type="list"  label="Upload all data files wich are data table" help= "Input a data collection."/>
```

Comment:
Could this be a normal data type param with multiple=true?

Reply:
I prefer use it like that for now and we will reevaluate this! THANK YOU!